ledger/sandbox-classic/src/test/lib/scala/platform/sandbox/auth/ServiceCallAuthTests.scala

```diff
@@ -141,4 +146,11 @@ trait ServiceCallAuthTests
   protected val canReadAsAdminRandomParticipantId: Option[String] =
     Option(customTokenToHeader(forParticipantId(UUID.randomUUID.toString, adminToken)))
 
+  protected def createUserAsAdmin(userId: String): Future[(User, Option[String])] = {
```

Comment:
This name is a bit misleading. It could be interpreted that the newly created user is an admin. Maybe createUserThroughAdmin or createUserByAdmin would be better?

Reply:
thx